commands/money/balance.py

```diff
@@ -0,0 +1,42 @@
+# /commands/money/balance.py
+import discord
+from discord.ext import commands
+from typing import Literal
+from common.database.decorator import database_connect
+from commands.common.functions import returnUserEntity, returnServerToUsersEntity
+
+
+# Define the slash command
+@database_connect
+async def balance_command(interaction: discord.Interaction, session=None):
+    discord_id = str(interaction.user.id)
+    server_id = str(interaction.guild.id)
+
+    # checks that the user exists
+    user = returnUserEntity(discord_id=discord_id, session=session)
+
+    # checks that the servertouser exists
+    user_entry = returnServerToUsersEntity(discord_id=discord_id, server_id=server_id, session=session)
+
+    # embedded message
+    embed = discord.Embed(
```

Comment:
Would it be worth making an embed function so we dont have to constantly type discord.embed(...)? Although we are still having to type all the content

Reply:
We could, although my only concern would be that it could heavily limit how much we can customize the embed or whether we embed it or not. Since that's just a specific way of displaying the bot's message and is hard to limit to specific parameters.